Source/controls/keymapper.hpp

```diff
@@ -0,0 +1,77 @@
+#pragma once
```

Comment:
Either this file uses LF line endings or GitHub fixed their suggestion feature, could you please check that it's CRLF.